specification/resources/resource-manager/Microsoft.Authorization/stable/2021-06-01/policyAssignments.json

```diff
@@ -0,0 +1,980 @@
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "PolicyAssignments"
+        ],
+        "operationId": "PolicyAssignments_Get",
+        "summary": "Retrieves a policy assignment.",
+        "description": "This operation retrieves a single policy assignment, given its name and the scope it was created at.",
+        "x-ms-examples": {
+          "Retrieve a policy assignment": {
+            "$ref": "./examples/getPolicyAssignment.json"
+          },
+          "Retrieve a policy assignment with a system assigned identity": {
+            "$ref": "./examples/getPolicyAssignmentWithIdentity.json"
+          },
+          "Retrieve a policy assignment with a user assigned identity": {
+            "$ref": "./examples/getPolicyAssignmentWithUserAssignedIdentity.json"
+          }
+        },
+        "parameters": [
+          {
+            "name": "scope",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The scope of the policy assignment. Valid scopes are: management group (format: '/providers/Microsoft.Management/managementGroups/{managementGroup}'), subscription (format: '/subscriptions/{subscriptionId}'), resource group (format: '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}', or resource (format: '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{resourceProviderNamespace}/[{parentResourcePath}/]{resourceType}/{resourceName}'",
+            "x-ms-skip-url-encoding": true
+          },
+          {
+            "name": "policyAssignmentName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the policy assignment to get."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns information about the policy assignment.",
+            "schema": {
+              "$ref": "#/definitions/PolicyAssignment"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "PolicyAssignments"
+        ],
+        "operationId": "PolicyAssignments_Update",
+        "summary": "Updates a policy assignment.",
+        "description": " This operation updates a policy assignment with the given scope and name. Policy assignments apply to all resources contained within their scope. For example, when you assign a policy at resource group scope, that policy applies to all resources in the group.",
+        "x-ms-examples": {
+          "Create or update a policy assignment with a system assigned identity": {
```

Comment:
Examples for patch should just say "Update"

Reply:
fixed...  thx